package.json

```diff
@@ -38,7 +38,8 @@
     "typescript": "^2.7.0-dev.20171025"
   },
   "scripts": {
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "tsc": "tsc -p ."
```

Comment:
Did this slip in from another branch?

Reply:
nope. decided to add it. So we can `yarn tsc`